client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -48,12 +49,16 @@ class _ProtectYourselfState extends State<ProtectYourself> {
   @override
   Widget build(BuildContext context) {
     return PageScaffold(
-        title: S.of(context).protectYourselfTitle,
-        showShareBottomBar: false,
-        announceRouteManually: true,
-        body: [SliverList(delegate: SliverChildListDelegate(_buildCards()))]);
+      title: S.of(context).protectYourselfTitle,
+      showShareBottomBar: false,
+      announceRouteManually: true,
+      body: [_factContent != null ? _buildBody() : LoadingIndicator()],
+    );
   }
 
+  SliverList _buildBody() =>
+      SliverList(delegate: SliverChildListDelegate(_buildCards()));
```

Comment:
Please add trailing commas

Reply:
What do you mean exactly?

Reply:
```suggestion
      SliverList(
          delegate: SliverChildListDelegate(
               _buildCards(),
           ),
      );
```

Reply:
Maybe mean for the array in page scaffold that uses this...  Yah.  Wondering if that should be a 'children' and a 'body' alternative that takes a single child.

Reply:
Fixed (the commas)